<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteEdgeWindow</class>
 <widget class="QDialog" name="DeleteEdgeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete edge</string>
  </property>
  <widget class="QLineEdit" name="fromLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="toLine">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="delEdgeBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="toLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="windowTitleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delete edge</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fromLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
